app/src/main/java/com/tari/android/wallet/application/baseNodes/BaseNodes.kt

```diff
@@ -20,11 +20,16 @@ import com.tari.android.wallet.service.connection.TariWalletServiceConnection
 import io.reactivex.disposables.CompositeDisposable
 import org.apache.commons.io.IOUtils
 
+private const val REGEX_ONION = "(.+::[A-Za-z0-9 ]{64}::/onion3/[A-Za-z0-9]+:[\\d]+)"
+private const val REGEX_IPV4 = "(.+::[A-Za-z0-9 ]{64}::/ip4/[0-9]{1,3}[.][0-9]{1,3}[.][0-9]{1,3}[.][0-9]{1,3}/tcp/[0-9]{2,6})"
```

Comment:
Public key is a hexadecimal value so you should rather use [a-f0-9] than [a-z0-9]

Reply:
Because the release branch is outdated and in the development I use different regexes